routing/message/handlers/command/passthrough_handler.go

```diff
@@ -13,48 +13,38 @@
 package command
 
 import (
-	"fmt"
-	"strings"
-
+	"github.com/ThreeDotsLabs/watermill"
 	"github.com/ThreeDotsLabs/watermill/message"
 	"github.com/eclipse-kanto/azure-connector/config"
-	"github.com/eclipse-kanto/azure-connector/util"
 	"github.com/eclipse-kanto/suite-connector/connector"
-	"github.com/pkg/errors"
 )
 
 const commandPassthroughHandlerName = "command_passthrough_handler"
 
+// A simple command passthrough handler that forwards all cloud-to-device messages from Azure IoT Hub to local MQTT broker on a preconfigured topic.
+// TODO: to be extended in the future as part of issue #21
```

Comment:
Clean up this TODO, if required, it can be put as a comment to the related ticket.